lucille-core/src/main/java/com/kmwllc/lucille/core/Indexer.java

```diff
@@ -48,10 +49,11 @@ public void terminate() {
   public Indexer(Config config, IndexerMessageManager manager, String metricsPrefix) {
     this.manager = manager;
     this.idOverrideField = config.hasPath("indexer.idOverrideField") ? config.getString("indexer.idOverrideField") : null;
+    this.indexOverrideField = config.hasPath("index.indexOverrideField") ? config.getString("indexer.indexOverrideField") : null;
```

Comment:
first occurrence should be should be "indexer.indexOverrideField", not "Index.indexOverrideField"